frontend/pages/DashboardPage/cards/MDM/MDM.tsx

```diff
@@ -190,6 +190,7 @@ const Mdm = ({
                   isAllPagesSelected={false}
                   disableCount
                   disablePagination
+                  hideFooter
```

Comment:
RACHEL: We have a hideFooter property on the Table Container we’re constantly using if we’re on page zero and there’s no further pages, that should be built into TableContainer instead of trying to constantly define it as a prop.
EST: 2 pts

disablePagination and no footer should auto hide footer